### DISCUSSION TITLE
Help Required to start and run the project for the first time at local pc.

### DISCUSSION THREAD

**Discussion by D-I-J** [DigitalPebble/storm-crawler#1088] [Q&A] [answered]
Hi everyone.

can anyone please guide me how to install the software at a local pc.

I have read the read me, but i don't understand it fully(i get confused reading it even if understand the dependencies it mentions.).

- [ ] I have installed the required jdk according to the read me as it was the simplest for me to understand.
- [ ] I have installed the required maven according to the read me as well.

Please Guide me.

I think I may be ask too much but I need to start to do any thing else.

**Comment by rzo1** [comment, marked as answer]
Maybe if you haven't done so already, look at the tutorials on https://www.youtube.com/playlist?list=PLiqxzwp5B4ZnvBEsdTwSHVJ5FRDTlCl9V

They are about Elasticsearch and probably about an older version of SC but they should help you understand some of the concepts a bit better.

**Reply by D-I-J** [reply]
Thanks for you response and giving time to my question.
The content look too hard for my level right now.
